src/core/src/GltfUtil.cpp

```diff
@@ -317,21 +317,19 @@ const CesiumGltf::ImageCesium& getImageCesium(const CesiumGltf::Model& model, co
 }
 
 std::pair<std::string, uint64_t> parseAttributeName(const std::string& attributeName) {
-    const auto regex = std::regex("^([a-zA-Z_]*)(_([1-9]\\d*|0))?$");
-
-    std::string semantic;
-    std::string setIndex;
-
-    std::smatch matches;
+    std::string semantic = "";
+    uint64_t setIndexU64 = 0;
 
-    if (std::regex_match(attributeName, matches, regex)) {
-        semantic = matches[1];
-        setIndex = matches[3];
+    if (attributeName[0] != '_') {
+        semantic = attributeName;
+    } else {
+        auto lastUnderscorePosition = attributeName.rfind('_');
+        semantic = attributeName.substr(0, lastUnderscorePosition);
+        auto setIndex =
+            attributeName.substr(lastUnderscorePosition + 1, attributeName.size() - lastUnderscorePosition + 1);
+        std::from_chars(setIndex.data(), setIndex.data() + setIndex.size(), setIndexU64);
     }
```

Comment:
I'm not sure if this check is robust enough. Some of the attributes below will work, but some will fail:

_FEATURE_ID_1
_CESIUMOVERLAY_1
TEXCOORD_1
POSITION
RANDOM_PROPERTY

A better solution might be to traverse backwards like, stopping either when:
* You hit an A-Z character (no set index)
* You hit an underscore (set index determined by preceding digits)

Reply:
Ah, good list. Is there a master list available? I assumed anything with a set index would start with '_', but it looks like TEXCOORD might not.

Reply:
This the official glTF list:

![image](https://github.com/CesiumGS/cesium-omniverse/assets/915398/5c92ae30-ffab-4a06-8fc6-711a20ee46bf)

But custom semantics are allowed too (anything starting with an underscore)

Reply:
Good resource. Algorithm updated to meet the Khronos spec.